Champions.ts

```diff
@@ -0,0 +1,35 @@
+import { Player } from "./models/Types";
+
+
+function findChampions(players: Player[]): Player[] {
+
+  if (players.length < 2) return players;
+
+
+  const championsList: Player[] = [];
+  const maxEloRatingByAge: number[] = [];
+
+  for (const player of players) 
+    maxEloRatingByAge[player.age] = Math.max(maxEloRatingByAge[player.age] || -1, player.eloRating);
```

Comment:
it's usually good to push parts in private methods, like here you could have a `computeMaxEloRatingByAge(players)` helper.

-> it helps separate the concerns, and makes the main function be more readable by reading "as english", and then you can dive in the implementation details if you need

Reply:
then the maxEloRatingByAge is not the right type, it should be `(number | undefined)[]` because all ages are not necessarily represented.

you don't have a bug because, line 20, `player.eloRating <= undefined` is false, but that's by luck.